### PR TITLE
Edit and consolidation pass of dev docs

### PR DESCRIPTION
**Pull request by DanVanAtta**
Combined the relatively newer and shorter dev docs to README, then did a pruning/edit pass to update and reduce text

### CODE REVIEW

File: docs/dev/README.md [hunk 1]
```diff
@@ -1,36 +1,101 @@
-### Git
+# Building and Running the Code
 
-TripleA repositories may make use of [Git Large File Storage](https://git-lfs.github.com/) (LFS).  In order to work with such a repository, devs will need to have a Git LFS client installed.  Installation instructions for various platforms is available [here](https://github.com/git-lfs/git-lfs#getting-started).
+```
+./gradlew test
```
Comment (RoiEXLab) [inline docs/dev/README.md]: `./gradlew check`

File: docs/dev/README.md [hunk 2]
```diff
@@ -1,36 +1,101 @@
...
+Before merge:
+- all comments questions should be answered/addressed
+
+### Code Review Ordering
+- Reviewers should start with the oldest PRs and work their way forward in time. Favor merging things in that order as well. The reason for the merge order is to make merge conflicts a bit more predictable, if you are the first PR in the queue, then there should be no merge conflicts. If 2nd, then in theory you would only have to worry about the one open PR request to cause merge conflicts.
+
+
+# Versioning 
+
+Simplified 3 number versioning system based on semantic versioning:
```
Comment (RoiEXLab) [inline docs/dev/README.md]: Well this is not really accurate :/

File: docs/dev/README.md [hunk 3]
```diff
@@ -1,36 +1,101 @@
...
+* Last discussed in: https://github.com/triplea-game/triplea/issues/1739
+
+# Troubleshooting
 
-## Set up IDE Formatter
-See [code format](https://github.com/triplea-game/triplea/blob/master/docs/dev/code_format.md)
+Response to a frozen game problem, below are steps notes of what would be needed to debug such a problem:
 
-## Useful Tools
+  - launch the game from a console window, running: ./triplea_unix.sh  | tee output.txt
```
Comment (ssoloff) [inline docs/dev/README.md]: I don't believe the _triplea_unix.sh_ script exists anymore.

File: docs/dev/code_format.md [hunk 4]
```diff
@@ -1,27 +0,0 @@
-## IDE Code Format Setup
-
-### Eclipse
-
-Project preferences > Java > java format, import and select the eclipse format xml file.
-
-- [https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea_java_eclipse_format_style.xml](https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea_java_eclipse_format_style.xml)
-- [https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea_java_eclipse_cleanup.xml](https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea_java_eclipse_cleanup.xml)
-- [https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea.importorder](https://github.com/triplea-game/triplea/blob/master/eclipse/format/triplea.importorder)
-
```
Comment (ssoloff) [inline docs/dev/code_format.md]: I don't see information related to where one can find the code formatter configuration files (under _eclipse/format/_) in the README.  There is a short blurb in _code_standards.md_, but it points to a broken link.  Somehow, we should note where these files reside.

File: docs/dev/README.md (hunk 3 above)
Comment (DanVanAtta) [reply docs/dev/README.md]: yeah, these notes are linux specific, not the best : |
Largely this is moved documentation, but it makes sense to go ahead and spend some time to prune, I removed the script mention in: d00b938

File: docs/dev/README.md (hunk 1 above)
Comment (DanVanAtta) [reply docs/dev/README.md]: that's an important command, the original docs had 'test' and 'check' command listed in other places. I updated the branch to have the check command included. It's a small decision to remove 'test' completely, that is not in scope of these updates as we are first consolidating, and then incrementally removing the out of date.

Comment (DanVanAtta) [reply docs/dev/README.md]: 4b2de734ac58af2caefb224eada04ed2e6b7e68f

File: docs/dev/code_format.md (hunk 4 above)
Comment (DanVanAtta) [reply docs/dev/code_format.md]: Fair enough, added a link to the section in: 188dec6a8164a4b254e9cf8d384c5535bebd58af

I reduced the verbosity, it's pretty sufficient to give a link to the folder. I hope/expect for us to clean up the IDE configuration files in the future, while here it's good to reduce the coupling between documentation and the folder structure. (The current top level has been messy for a very long time)

**Comment by DanVanAtta** [comment]
@ssoloff @RoiEXLab while taking another pass there were more opportunities to consolidate and prune content. All but one comment should be addressed as well. The versioning stuff is a documentation of the direction we are heading to @RoiEXLab , so it can't be readily changed without some dedicated discussion (and that in specific is out of scope for the purpose of this update).

I'm willing to do maybe one more pass, but at some point this should just get merged so the scope does not grow further (again, trying to consolidate and reduce content primarily but not get bogged down in corrections at this moment).

**Comment by DanVanAtta** [comment]
@RoiEXLab @ssoloff please have a final look-over if you would so we can close this iteration.

**Pull request merged**